src/AutorestSwift/Utils/String+Extensions.swift

```diff
@@ -77,23 +77,23 @@ extension String {
     }
 
     /// Converts a string into one with camel casing. Acronyms are forced to camel case as well.
-    var toCamelCase: String {
-        let components = splitAndJoinAcronyms
-        var casedComponents = [String]()
-        for (index, comp) in components.enumerated() {
-            casedComponents.append(index == 0 ? comp : comp.capitalized)
-        }
-        let value = casedComponents.joined()
-        return value
-    }
+//    var toCamelCase: String {
+//        let components = splitAndJoinAcronyms
+//        var casedComponents = [String]()
+//        for (index, comp) in components.enumerated() {
+//            casedComponents.append(index == 0 ? comp : comp.capitalized)
+//        }
+//        let value = casedComponents.joined()
+//        return value
+//    }
 
     /// Converts a string into one with camel casing. Acronyms are forced to camel case as well.
-    var toPascalCase: String {
-        let components = splitAndJoinAcronyms
-        let casedComponents = components.map { $0.capitalized }
-        let value = casedComponents.joined()
-        return value
-    }
+//    var toPascalCase: String {
```

Comment:
Maybe just deleted the code instead of commented out?

Reply:
Removed.